package.json

```diff
@@ -15,7 +15,8 @@
     "cordova:platform"
   ],
   "scripts": {
-    "test": "npm run eslint && npm run test:coverage",
+    "cover": "npm run test:coverage",
```

Comment:
What is the purpose of a new script that calls another script and does nothing more?